### DISCUSSION TITLE
Ordering at JLCPCB. Problem with MMBT3906

### DISCUSSION THREAD

**Discussion by nullsumme** [rbaron/b-parasite#203] [Q&A] [answered]
I'm trying to order some PCBs from JLCPCB. 
I get the following error:
<img width="1379" alt="Bildschirmfoto 2024-05-30 um 20 37 14" src="https://github.com/rbaron/b-parasite/assets/369577/aaecfcc0-4477-4a9c-a945-a38a27bfcaef">
Does that mean that the MMBT3906 is not in stock?

**Comment by deimi** [comment, marked as answer]
yes, but only this specific part number. You can choose another MMBT3906 from a different vendor like this one C2984759 https://jlcpcb.com/partdetail/Fuxinsemi-MMBT3906/C2984759

**Comment by rbaron** [comment]
@nullsumme yes, @deimi is correct. Usually there are 1:1 parts that are in stock instead. It still worth it double checking the parts' pinouts, just in case.
